Automattic/wp-calypso
# Plugins: remove the manage/plugins/cache feature

The `manage/plugins/cache` feature flag is disabled in all configurations and seems to have been that way for 2 years, since the the initial OSS commit.

This PR removes the feature, together with the dead code that implements it.

The feature has been caching the lists of site plugins and lists or WP.org plugins in the plugin browser in local storage. Something similar to state serialization we have in Redux.

**To test:**
Go to Plugins Browser and try to click on various lists (all, featured, new, popular) and search. Are plugins loaded from WP.org API as expected?

Go to Plugins Manage page for a Jetpack/Atomic site. Is the plugin list loaded and displayed as expected?
